src/components/ReadWriteFirestore.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from 'react';
+import { collection, addDoc, getDocs } from 'firebase/firestore';
+import { db } from '../lib/firebase';
+
+function ReadWriteFirestore() {
+  const [state, setState] = useState(0);
+  const [user, setUser] = useState([]);
```

Comment:
As you likely know, 'naming is hard' ™️  but it also allows you to convey a lot of meaning/intention when coding. In this instance, I found myself a little lost at first reading the code as this app will be generating a `List` of `items` so the use of `user` through me off a bit.

This is by no means a stopper for this PR and can definitely be addressed in later work as you build out the App.